x/auth/client/cli/cli_test.go

```diff
@@ -1024,6 +1024,61 @@ func (s *IntegrationTestSuite) createBankMsg(val *network.Validator, toAddr sdk.
 	return res
 }
 
+func (s *IntegrationTestSuite) TestSignWithMultiSigners() {
+	val0, val1 := s.network.Validators[0], s.network.Validators[1]
+	val0Coin := sdk.NewCoin(fmt.Sprintf("%stoken", val0.Moniker), sdk.NewInt(10))
+	val1Coin := sdk.NewCoin(fmt.Sprintf("%stoken", val1.Moniker), sdk.NewInt(10))
+	_, _, addr1 := testdata.KeyTestPubAddr()
+
+	// Creating a tx with 2 msgs from 2 signers: val0 and val1.
+	txBuilder := val0.ClientCtx.TxConfig.NewTxBuilder()
+	txBuilder.SetMsgs(
+		banktypes.NewMsgSend(val0.Address, addr1, sdk.NewCoins(val0Coin)),
+		banktypes.NewMsgSend(val1.Address, addr1, sdk.NewCoins(val1Coin)),
+	)
+	txBuilder.SetFeeAmount(sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))))
+	txBuilder.SetGasLimit(testdata.NewTestGasLimit())
+	// Set signer_infos for both signers. Note: we use the empty signature hack.
+	// TODO
+	s.Require().Equal([]sdk.AccAddress{val0.Address, val1.Address}, txBuilder.GetTx().GetSigners())
+
+	// Write the unsigned tx into a file.
+	txJSON, err := val0.ClientCtx.TxConfig.TxJSONEncoder()(txBuilder.GetTx())
```

Comment:
Since I didn't add any signer_info (see TODO on line 1042), that array is empty:

```js
// txJSON
{
  "body": {
    "messages": [
      {
        "@type": "/cosmos.bank.v1beta1.MsgSend",
        "from_address": "cosmos1mr3t7jx3mwrl47zn7vanxvp9gtqj6td4nfmexu",
        "to_address": "cosmos1yj0nu9kpmt0feqsut4jffegj9uhucf00n8hx3e",
        "amount": [{ "denom": "node0token", "amount": "10" }]
      },
      {
        "@type": "/cosmos.bank.v1beta1.MsgSend",
        "from_address": "cosmos12n3xrjtpqmayjjce7scruxc5d99zvjdamjmjda",
        "to_address": "cosmos1yj0nu9kpmt0feqsut4jffegj9uhucf00n8hx3e",
        "amount": [{ "denom": "node1token", "amount": "10" }]
      }
    ],
    "memo": "",
    "timeout_height": "0",
    "extension_options": [],
    "non_critical_extension_options": []
  },
  "auth_info": {
    "signer_infos": [],
    "fee": {
      "amount": [{ "denom": "stake", "amount": "10" }],
      "gas_limit": "100000",
      "payer": "",
      "granter": ""
    }
  },
  "signatures": []
}
```